src/rgw/setup/rgw_service.py

```diff
@@ -31,14 +31,24 @@ def start(conf: MappedConf, config_file, log_file, motr_trace_dir, index: str =
         try:
             os.environ['M0_TRACE_DIR'] = motr_trace_dir
             cmd = "/usr/bin/radosgw"
-            args = f"-f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
-            args = shlex.split(args)
+            args = f" -f --name client.rgw-{index} -c {config_file} --no-mon-config 2>1"
+            # add timestamp to each line in from previous command
+            time_stamp = """ | gawk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }' """
+            # redirect the output to both stdout and log_file
+            redirect = f" | tee {log_file} "
+            final_cmd = cmd + args + time_stamp + redirect
             sys.stdout.flush()
             sys.stderr.flush()
-            os.execl(cmd, cmd, *args)  # nosec
+            bash = '/bin/bash'
+            cmd_file = 'radosgw_start.sh'
+            with open(cmd_file,'w') as fd:
+                fd.write('set -o pipefail ;')
+                fd.write(final_cmd + ';')
```

Comment:
comment here on why command not directly executed using execl() and through file.
Something like below, but add all reasoning here
"Final command has multiple piped commands, which is not supported directly os.execl()"